libs/application/core/src/types/ApplicationTypes.ts

```diff
@@ -12,16 +12,49 @@ export enum ApplicationTypes {
   PARTY_LETTER = 'PartyLetter',
 }
 
-export const ApplicationTranslations = {
-  ExampleForm: 'example.application',
-  Passport: null,
-  DrivingLessons: 'dl.application',
-  DrivingLicense: 'dl.application',
-  ParentalLeave: 'pl.application',
-  MetaApplication: null,
-  DocumentProviderOnboarding: 'dpo.application',
-  HealthInsurance: 'hi.application',
-  ChildrenResidenceChange: 'crc.application',
-  DataProtectionAuthorityComplaint: 'dpac.application',
-  PartyLetter: 'ple.application',
+export const ApplicationConfigurations = {
+  [ApplicationTypes.EXAMPLE]: {
+    slug: 'example',
+    translation: 'example.application',
+  },
+  [ApplicationTypes.PASSPORT]: {
+    slug: 'vegabref',
+    translation: 'pp.application',
+  },
+  [ApplicationTypes.DRIVING_LESSONS]: {
+    slug: 'okunam',
+    translation: 'dl.application',
+  },
+  [ApplicationTypes.DRIVING_LICENSE]: {
+    slug: 'okuskilriki',
```

Comment:
`okuskirteini`

Reply:
I checked with the team who worked on it, they told me `okuskilriki` was correct

Reply:
[https://island.is/okuskirteini](https://island.is/okuskirteini)

🤷